addons/event/models/event_mail.py

```diff
@@ -25,21 +25,21 @@
     'now': lambda interval: relativedelta(hours=0),
 }
 
+class MailValues(NamedTuple):
+    interval_nbr: int
+    interval_unit: str
+    interval_type: str
+    template_ref: str
 
 class EventTypeMail(models.Model):
     """ Template of event.mail to attach to event.type. Those will be copied
     upon all events created in that type to ease event creation. """
     _name = 'event.type.mail'
     _description = 'Mail Scheduling on Event Category'
 
-    @api.model
-    def _selection_template_model(self):
-        return [('mail.template', 'Mail')]
-
     event_type_id = fields.Many2one(
         'event.type', string='Event Type',
         ondelete='cascade', required=True)
-    notification_type = fields.Selection([('mail', 'Mail')], string='Send', default='mail', required=True)
```

Comment:
Note that we could probably keep a computed field to have directly access to the notification type. It would allow to avoir "if scheduler.template_ref._name == 'sms.template'". We still have to do some computation based on type so having a field for that is useful. However it would be a computed one (not stored), present for code, not in UI. What do you think ?